src/webapp/pages/NeighborhoodFinder.js

```diff
@@ -85,8 +91,9 @@ export default function NeighborhoodFinder() {
   };
 
   return (
-    <Box className={classes.root}>
+    <Box className={classes.root + (minimalView ? " " + classes.rootMinimalView : "")}>
```

Comment:
you can use string templates here! e.g. `${classes.root} ${minimalView ? classes.rootMinimalView  : '""}`

Reply:
I'll make this change :+1: . Note that there will be an extraneous space now, though.

Reply:
oh actually we should be using the classnames lib here. maybe for a future PR? but i think it's an acceptable thing to have extra whitespace since functionality is affected.

Reply:
Updated!

Reply:
+1 for classnames, +1 for a future PR :~). The extra space doesn't affect a thing here, so no worries at all.